pkg/operator/controller/controller.go

```diff
@@ -189,15 +191,19 @@ func (r *reconciler) enforceEffectiveIngressDomain(ci *ingressv1alpha1.ClusterIn
 		return nil
 	}
 
+	updated := ci.DeepCopy()
```

Comment:
Just a random readability thing to consider next time... would it be more clear to reassign ci like

```go
ci = ci.DeepCopy()
```

Reply:
I think you talked me out of the need for using copies anyway, if you want to just revert these changes to reduce the overall diff set... if you want to keep this that's fine too

Reply:
> would it be more clear to reassign ci like
> 
> ```go
> ci = ci.DeepCopy()
> ```

Wouldn't that change the pointer value so that the `r.Client.Get` would no longer be updating the caller's `ci`?